### ISSUE TITLE
Advice on installing on an old Quest 2 without updating

### ISSUE THREAD

**Issue by AtlasC0R3** [basti564/Oculess#126]
My apologies if this issue is outside the scope of this project, as it is more about asking for advice on using this project in a likely very unusual situation. If this is not the right place to ask, please indicate where else I may find answers.

I have a Quest 2 from an acquaintance that has not been turned on or updated for a while (likely ~2 years), as such, it is probably outdated and likely does not have the developer mode turned on.
How would I go about enabling developer mode on the device, then installing Oculess, all while preventing updates (that will render Oculess useless/incompatible)? Should I manually flash an update up to v41 perhaps for better game compatibility?
Or is it somehow possible to update the device, enable developer mode, then downgrade where Oculess is relevant?

**Comment by unohara-fuukei** [comment]
if you are trying to improve the compatibility by update to ver 41, i can assume that the target games still won't work 'cause there is merely difference in the series of system those before v51 -- android target api 26. And once update to lastest OS you  gonna lose the compatibility of bunch of games. 
It's not hard to update a outdated system by sideload/ official update, but for game resources you have to make a vigilant decision.
by the way, i met the completely same condition these days and choose to make a official update(it directly leads to v71), and the new system got device battery consume faster than the former.

**Comment by AtlasC0R3** [comment]
I'll be honest, I gave up and factory reset the thing. Made a hotspot through mobile data, updated the thing, burner underage Meta account (to mitigate data collection), developer mode with an unused prepaid card, only sideload *cough cough* "acquired" games, so on.
The one crucial thing I did, though, and I'd really like to share this, is activate DNS-over-HTTPS via Android's system settings to pass through Mullvad's ad-blocking DNS, as it basically prevents any and all Meta/Oculus connection through the Internet AFAIK.

I guess this issue is no longer relevant now.
